testdata/serviceconfig.yml

```diff
@@ -32,7 +32,7 @@ rules:
     params:
       logName: "access_log"
       log:
-        logFormat: 1 # COMMON
+        descriptorName: "common"
```

Comment:
`descriptor_name` is consistent with all the other aspects (including application logs). I think it'd be pretty odd to make this one aspect different.